lib/passenger.rb

```diff
@@ -16,6 +18,26 @@ def add_trip(trip)
       @trips << trip
     end
 
+    def net_expenditures
+      total = 0
+      @trips.each do |trip|
+        total += trip.cost unless trip.end_time.nil?
+      end
+
+      return total
+
+    end
+
+    def total_time_spent
+      total = 0
+      @trips.each do |trip|
+        total += trip.trip_duration
+      end
+
+      return total
```

Comment:
You can simplify this using `sum`:

```suggestion
      return @trips.sum(&:trip_duration)
```